events/_event-landing-macros.html

```diff
@@ -10,19 +10,29 @@
 
    post: A post from a query result.
 
+   media_object: The optional boolean flag to wrap the block in a media object
+                 and include a map image.
+                 Defaults to True.
+
    ========================================================================== #}
 
-{% macro event_preview(post) %}
+{% macro event_preview(post, media_object=True) %}
```

Comment:
Per note for true/false on http://jinja.pocoo.org/docs/dev/templates/#literals looks like this should actually be `media_object=true`